### PR TITLE
Docs: features

### PR DESCRIPTION
**Pull request by carwack**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been reviewed and added / updated if needed (for bug fixes
      /start features)

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] Documentation content changes
- [ ] Other (please describe):

## What is the new behavior?

- Add docs under feature topic

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by carwack** [comment]
@codebender828 The doc page for Global Styles is missing (https://chakra-ui.com/docs/features/global-styles) but I'm not sure how this is implemented in the Vue version. How is this done?

And if you have time, could you please review the docs so it's 100% correct. Thanks! 🙏

File: docs/pages/features/chakra-factory.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Chakra Factory
+
+Chakra factory serves as an **object of chakra enabled JSX elements**, and also
```
Comment (codebender828) [inline docs/pages/features/chakra-factory.md:3]:
```suggestion
Chakra factory serves as an **object of chakra enabled HTML elements**, and also
```

File: docs/pages/features/chakra-factory.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+<chakra.button
+  px="3"
+  py="2"
+  bg="green.200"
+  rounded="md"
+  :_hover="{ bg: "green.300" }"
+>
+  Click me
+</chakra.button>
+```
```
Comment (codebender828) [inline docs/pages/features/chakra-factory.md:27]: I think we can directly use Vue SFCs for the docs codeblocks

```suggestion
```vue
<chakra.button
  px="3"
  py="2"
  bg="green.200"
  rounded="md"
  :_hover="{ bg: 'green.300' }"
>
  Click me
</chakra.button>
```
```

File: docs/pages/features/chakra-factory.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+```jsx
+<chakra.h1 fontSize="lg"> Heading </chakra.h1>
+```
+
+## Chakra factory function
+
+This is a function that converts **non-chakra components** or **jsx element** to
+chakra-enabled components so you can pass style props to them.
+
+Consider a package called `my-awesome-components`, let's use the chakra factory
```
Comment (codebender828) [inline docs/pages/features/chakra-factory.md:43]:
```suggestion
Consider a component called `my-awesome-component`, let's use the chakra factory
```

File: docs/pages/features/chakra-factory.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+```jsx
+const ChakraAwesomeComponent = chakra(awesomeComponent, {
+  baseStyle: {
+    bg: "papayawhip",
+    color: "red.500"
+  },
+})
+```
+
+You can also use the chakra factory on jsx-elements as well.
```
Comment (codebender828) [inline docs/pages/features/chakra-factory.md:79]:
```suggestion
You can also use the chakra factory on HTML elements as well.
```

File: docs/pages/features/color-mode.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
+# Color Mode
```
Comment (codebender828) [inline docs/pages/features/color-mode.md:1]: I think we should mark this page as a WIP. I'm still considering how to best provide the color modes in the template. But for now, we can use the `useChakra` hook.

```ts
import { useChakra } from '@chakra-ui/vue-next'

const { colorMode } = useChakra()

// You can also use the `useColorMode` hook
import { useColorMode } from '@chakra-ui/vue-next'

const { colorMode, toggleColorMode } = useColorMode()

```

**Review by codebender828 — CHANGES_REQUESTED** [review]
Thanks for this. Everything is mostly okay. Just need to make a few changes

File: docs/pages/features/color-mode.md (hunk 5 above)
Comment (carwack) [reply docs/pages/features/color-mode.md:1]: Should we just put a WIP message at the top and remove what is currently inside the doc file? And suggest using the `useChakra` hook?

**Review by codebender828 — APPROVED** [review]

**Pull request merged**